tests/integration/test_eoc_journal.py

```diff
@@ -393,10 +437,23 @@ def test_link_to_pb_answers_pdf_not_displayed_when_none_available_in_student_vie
             self.get_element_for_pb_answers_pdf_report
         )
 
-    def test_progress_metrics_listed_in_student_view(self):
-        self.set_standard_scenario()
-        self.go_to_view('student_view')
-        self.fix_js_environment()
+    def test_metrics_hidden_by_default(self):
```

Comment:
Nit: rename to `test_progress_metrics_hidden_by_default` for consistency.

Reply:
I renamed it to `test_all_metrics_hidden_by_default`, actually as it tests whether progress, proficiency and engagement metrics are hidden by default.